### ISSUE TITLE
Doubt about the Hamiltonian in Challenge 3 (Many body localization)

### ISSUE THREAD

**Issue by ruihao-li** [qiskit-community/ibm-quantum-spring-challenge-2022#14]
In Challenge 3 which investigates the many-body localization phenomenon, the tight-binding Hamiltonian used was $H_\text{tb}/\hbar = \sum_i(X_i X_{i+1} + Y_i Y_{i+1}) + \sum_i \epsilon_i Z_i$. However, this Hamiltonian does *not* include particle interaction. It should be evident by doing a Jordan-Wigner transformation to map it to a fermionic Hamiltonian, which leads to $H_\text{tb}/\hbar = \sum_i (c_i^\dagger c_{i+1} + \text{h.c.}) + \sum_i \epsilon_i c_i^\dagger c_i$. This is a *free fermion* Hamiltonian because there is no two-body interaction term like $n_i n_{i+1}$ present, where $n_i = c_i^\dagger c_i$. 

In fact, the paradigmatic spin-chain Hamiltonian used to study many-body localization in the literature is the Heisenberg model. One example would be the XXZ model, where an additional term $\sum_i Z_i Z_{i+1}$ is included to give exactly the two-body interaction term stated above. Under the section "Many-body quantum dynamics" in the Challenge notebook, it is stated that "Under $H_\text{tb}$, each site can only be occupied by a single particle, resulting in particle repulsion interaction". But a priori, there is nothing forbidding two particles occupying the same site under such Hamiltonian (without interaction), I think.

So in summary, I think the Hamiltonian used in this problem does not include particle-particle interaction and hence is not suited for studying many-body localization. Everything we saw in this Challenge problem is still the result of Anderson localization. Please correct me if I am wrong.
